data/human/culture conversations.txt

```diff
@@ -514,7 +514,7 @@ mission "News Reporter - Paradise"
 			`	The reporter frowns. "Can you explain your answer for our viewers?"`
 			choice
 				`	"The interests of every world should be considered, whether we agree or not."`
-				`	"The Free Worlds aren't represented well enough to gain traction through normal channels."`
+				`	"The Free Worlds isn't represented well enough to gain traction through normal channels."`
```

Comment:
I think this works better as a plural. It is referring to the worlds that make up the Free Worlds, not the organisation as a whole.

Reply:
```suggestion
				`	"The Free Worlds aren't represented well enough to gain traction through normal channels."`
```